docs/src/app/(public)/(content)/react/components/combobox/demos/hero/css-modules/index.tsx

```diff
@@ -0,0 +1,289 @@
+import * as React from 'react';
+import { Combobox } from '@base-ui-components/react/combobox';
+import styles from './index.module.css';
+
+export default function ExampleCombobox() {
+  const [inputValue, setInputValue] = React.useState('');
+  const [value, setValue] = React.useState('');
+
+  const filteredCountries = React.useMemo(() => {
+    if (inputValue.trim() === '') {
+      return countries;
+    }
+    return countries.filter((country) =>
+      country.name.toLowerCase().includes(inputValue.toLowerCase()),
+    );
+  }, [inputValue]);
+
+  return (
+    <Combobox.Root
+      value={value}
+      onValueChange={(nextValue) => {
+        setValue(nextValue);
+        React.startTransition(() => setInputValue(nextValue));
```

Comment:
No idea why but I noticed `React.startTransition` (or `React.useDeferredValue`) can break, but only with Fast Refresh(?) after editing the files. It ends up not updating the value.

Not sure if it's a React, Next.js, MDX, the demo, or source code bug causing it though